mpm/src/main.py

```diff
@@ -24,6 +24,8 @@
 now = datetime.datetime.now()
 current_year = now.year
 current_month = now.month
+version_crawler = os.getenv('GIT_COMMIT')
```

Comment:
Pode checar se essa variável é vazia e dar um erro bem descritivo se for, por favor?

Reply:
@danielfireman, chequei ela sim, é um hash correspondente ao ultimo commit. Mas vou acrescentar tbm uma exceção caso seja vazia